### ISSUE TITLE
Random ConcurrentModificationException Crashes

### ISSUE THREAD

**Issue by 123FLO321** [PrototypeTrousers/Applied-Energistics-2#184]
**Describe the bug**
When running for a while large ME systems will eventually crash the server.
```
java.util.ConcurrentModificationException
    at appeng.util.item.ItemList$ChainedIterator.next(ItemList.java:191)
    at appeng.util.item.ItemList$ChainedIterator.next(ItemList.java:166)
    at appeng.me.cluster.implementations.CraftingCPUCluster.storeItems(CraftingCPUCluster.java:760)
    at appeng.me.cluster.implementations.CraftingCPUCluster.updateCraftingLogic(CraftingCPUCluster.java:545)
    at appeng.me.cache.CraftingGridCache.onUpdateTick(CraftingGridCache.java:127)
    at appeng.me.GridCacheWrapper.onUpdateTick(GridCacheWrapper.java:40)
    at appeng.me.Grid.update(Grid.java:229)
    at appeng.hooks.TickHandler.onTick(TickHandler.java:182)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_529_TickHandler_onTick_TickEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
    at net.minecraftforge.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:266)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:712)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Thread.java:748)
 ```

**To Reproduce**
- Build complex ME system
- Many active craftings at once
- Eventually the server will crash

**Additional context**
[crash-2022-11-12_12.33.54-server.txt](https://github.com/PrototypeTrousers/Applied-Energistics-2/files/9994972/crash-2022-11-12_12.33.54-server.txt)

**Environment**
- Minecraft Version: 1.12.2
- AE2 Version: rv6-stable-7-extended_life-v0.54.11
- Forge Version: 14.23.5.2847

**Comment by PrototypeTrousers** [comment]
AE2FC issue. @GlodBlock might be able to update the transformed method to match the current behaviour

**Comment by GlodBlock** [comment]
fixed in 2.4.5, just updated to the latest in curseforge https://www.curseforge.com/minecraft/mc-mods/ae2-fluid-crafting-rework/files/4068625

**Comment by GlodBlock** [comment]
fixed in 2.4.7 @123FLO321

**Comment by 123FLO321** [comment]
thank you
